solution/ui/regulations/eregs-vite/src/main.js

```diff
@@ -11,8 +15,29 @@ const { customUrl, host } = mountEl.dataset;
 
 Vue.directive("clickaway", Clickaway);
 
+const router = vueRouter({ customUrl, host });
+
+// Silence duplicate navigation errors
+// see:
+// - https://stackoverflow.com/questions/57837758/navigationduplicated-navigating-to-current-location-search-is-not-allowed
+// - https://github.com/vuejs/vue-router/issues/2872
+const originalPush = router.push;
+router.push = function push(location, onResolve, onReject) {
+    if (onResolve || onReject) {
```

Comment:
Extends built in Vue router `push` method to silence duplicate navigation errors in the console.  We've been seeing these since the Resources page launched and it's time to squelch them.  See links above for discussion about these errors and why the code in this code block was the method to address these errors as chosen by the community.